### DISCUSSION TITLE
Target line: code translation

### DISCUSSION THREAD

**Discussion by ukraine-sdg** [open-sdg/open-sdg#1528] [Q&A] [answered]
Dear colleagues,

I have tried you recommendation about making target line for 2020.

Unfortunately, on the graph the code appears instead of "2020 indicator target" 

I have tried it for https://ukraine-sdg.github.io/site/1-1-1/

Here is my translation file: https://github.com/ukraine-sdg/data/blob/develop/translations/en/indicator.yml

Could you please help?

**Comment by brockfanning** [comment, marked as answer]
@ukraine-sdg In your translation file, instead of this...

```
indicator.annotation_2020_target: 2020 indicator target
```

...try this:

```
annotation_2020_target: 2020 indicator target
```

As an explanation, the "indicator" part actually comes from the name of the translation file, so it is not necessary to include it again.

**Reply by ukraine-sdg** [reply]
Thank you! It helped.

**Reply by LucyGwilliamAdmin** [reply]
Apologies, when copying over the field I accidentally included `indicator.`!
